storage_image_product/static/src/js/storage_image_product.js

```diff
@@ -0,0 +1,100 @@
+odoo.define('storage_image_product.image_handle', function (require) {
```

Comment:
@qgroulard Can't this widget be added on storage_image module level instead of product one ? In order to be reused.

Reply:
Arf, didn't read @sebastienbeau comment.